requirements/edx/base.txt

```diff
@@ -171,7 +171,7 @@ git+https://github.com/joestump/python-oauth2.git@b94f69b1ad195513547924e380d926
 oauthlib==3.0.1           # via -c requirements/edx/../constraints.txt, -r requirements/edx/base.in, django-oauth-toolkit, lti-consumer-xblock, requests-oauthlib, social-auth-core
 openedx-calc==1.0.9       # via -r requirements/edx/base.in
 git+https://github.com/edx/edx-ora2.git@2.6.25#egg=ora2==2.6.25  # via -r requirements/edx/github.in
-packaging==20.3           # via bleach, drf-yasg
+packaging==20.4           # via bleach, drf-yasg
```

Comment:
Take a look at this part of the diff—makes me pretty nervous: https://github.com/pypa/packaging/pull/283/files#diff-149972dc10abff58ee5c929a64b52c2eR151

Reply:
Looks like we wouldn't be affected by this; we don't use `packaging` directly, and bleach and drf-yasg only use it to retrieve their own version numbers.